atomic_reactor/constants.py

```diff
@@ -108,6 +108,8 @@
 PLUGIN_VERIFY_MEDIA_KEY = 'verify_media'
 PLUGIN_EXPORT_OPERATOR_MANIFESTS_KEY = 'export_operator_manifests'
 PLUGIN_PUSH_OPERATOR_MANIFESTS_KEY = 'push_operator_manifests'
+PLUGIN_BUILD_SOURCE_CONTAINER_KEY = 'build_source_container'
```

Comment:
Looking at this again, I don't think we should have the 'build' prefix in the plugin name. Comparing with orchestrate_build:
* it's a build plugin
* the plugin's name is orchestrate_build (line 101)
* the file is named build_orchestrate_build.py (${plugintype}_${pluginname}.py)
* the class implementing it is called OrchestrateBuildPlugin

Following that pattern we'd want:
* the plugin's name is source_container
* the file is named build_source_container.py
* the class would be named SourceContainerPlugin (or whatever, this part isn't so important)

Reply:
updated, thank you